### PR TITLE
Feat: My Accounts page

### PR DESCRIPTION
**Pull request by katspaugh**
## What it solves

A simplified version of #3176 w/o a Watchlist, fiat balances and queue counters. Only the new login page, the sidebar is not affected.

The Safes are sorted by current chain and if they are added or not.

## Screenshots
Screenshot 2024-02-09 at 14 00 32

## How to test it

* Go to the Welcome page
* Connect a wallet
* Press "Continue with Metamask"

If you aleady have owned/added Safes, it will show them.

### CODE REVIEW

File: src/components/welcome/MyAccounts/useAllSafes.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+      return uniqueAddresses.map((address) => ({
+        address,
+        chainId,
+        threshold: allAdded[chainId]?.[address]?.threshold,
+        owners: allAdded[chainId]?.[address]?.owners.length,
+      }))
+    })
+  }, [configs, allAdded, allOwned, currentChainId])
+}
```
Comment (katspaugh) [inline src/components/welcome/MyAccounts/useAllSafes.ts:44]: This hook needs tests.

Also, I'm not sure sorting by current chain + added Safes is the best way to do it, but given that most users have a small amount of safes, this probably doesn't matter much.

**Review by jmealy — APPROVED** [review]

**Comment by liliya-soroka** [comment]
Board - https://www.notion.so/safe-global/de2a3e284fd944cfa4d01076350159d8?v=8caccbde30724a448e5a44b77f9c0a8d

File: src/components/welcome/MyAccounts/CreateButton.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import Link from 'next/link'
+import { AppRoutes } from '@/config/routes'
+import { useCurrentChain } from '@/hooks/useChains'
+
+const CreateButton = () => {
+  const currentChain = useCurrentChain()
+
+  return (
+    <Link
+      href={{ pathname: AppRoutes.newSafe.create, query: { chain: currentChain?.shortName } }}
```
Comment (usame-algan) [inline src/components/welcome/MyAccounts/CreateButton.tsx:11]: Should we also track this button click? IMO it should be the new entry point for safe creation flow cc @alicecha

**Comment by liliya-soroka** [comment]
Verified

**Pull request merged**